Makefile

```diff
@@ -115,7 +115,7 @@ lint-go:
 lint: lint-go lint-js
 
 dump-test-schema:
-	go test -tags full,fts5,netgo -parallel 8 ./server/datastore/mysql/ -run TestMigrations
+	go run ./cmd/dbutils ./server/datastore/mysql/schema.sql
```

Comment:
I faced some weirdness when running all tests, so I moved the schema generation to a cmd. The logic is the same. The test remains, but without the file writing.

Reply:
Any concern with moving this under `./tools` and leaving `./cmd` for end-user commands?

Reply:
None! Done.

Reply:
I also updated the created_at/updated_at to make the schema generated idempotent since it didn't seem like a big change.